web/impact/requirements/base.txt

```diff
@@ -7,7 +7,7 @@ coreapi
 hiredis==0.2.0
 markdown # Markdown support for the browsable API.
 mysqlclient
-oauthlib==2.0.4
```

Comment:
I'd really like to see us get into the practice of pinning dependencies to explicit version numbers. Many times we've had to scramble to deal with problems when an automatic upgrade of a dependency has broken our application. 

I'm planning to make  a ticket on that subject, so this doesn't have to be a blocker for me on this PR, but I wanted to raise it -- especially given that an unpinned dependency was the cause of the original bug here ([AC-4989](https://masschallenge.atlassian.net/browse/AC-4989)).